game_eggs/steamcmd_servers/hlds_server/rehlds/egg-custom-re-h-l-d-s-engine-game.json

```diff
@@ -44,7 +44,7 @@
             "name": "Game Name",
             "description": "The name corresponding to the game to download and run using HLDS.\r\n\r\nall the HLDS server names are here - https:\/\/developer.valvesoftware.com\/wiki\/Dedicated_Server_Name_Enumeration",
             "env_variable": "HLDS_GAME",
-            "default_value": "cstrike",
```

Comment:
Why removing cstrike ? It is working with HLDS

Reply:
In my tests, cstrike was not being retreived automatically, and was producing errors. AppID 90 always has valve, regardless of what game mod you specify. There's actually been a history of SteamCMD not respecting the mod arguments occasionally, so it's probably better to default to valve anyway. In my opinion at least.